### PR TITLE
EVA-3438 - Lower log level

### PR DESCRIPTION
**Pull request by tcezard**
- Remove output of commands from log
- Demote most of the logging to debug and add general log message at info level
- Add file to log all debug message to

### CODE REVIEW

File: eva_sub_cli/__init__.py [hunk 1]
```diff
@@ -15,3 +15,13 @@
 SUBMISSION_WS_VAR = 'SUBMISSION_WS_URL'
 ENA_WEBIN_ACCOUNT_VAR = 'ENA_WEBIN_ACCOUNT'
 ENA_WEBIN_PASSWORD_VAR = 'ENA_WEBIN_PASSWORD'
+
+
+def is_submission_dir_writable(submission_dir):
```
Comment (apriltuesday) [inline eva_sub_cli/__init__.py:20]: I'm not a huge fan of putting actual functions/logic in `__init__.py`, can it maybe go in `main.py`?

File: eva_sub_cli/validators/validator.py [hunk 2]
```diff
@@ -54,7 +56,10 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.sub_config.backup()
         self.sub_config.write()
-
+    @staticmethod
+    def _quiet_commands(command_description, command, **kwargs):
```
Comment (apriltuesday) [inline eva_sub_cli/validators/validator.py:60]: The plural "commands" seems weird to me, maybe:
```suggestion
    def _run_quiet_command(command_description, command, **kwargs):
```

**Review by nitin-ebi — APPROVED** [review]

File: eva_sub_cli/__init__.py (hunk 1 above)
Comment (apriltuesday) [reply eva_sub_cli/__init__.py:20]: Utils is inevitable, embrace the utils :lotus_position: 

(I'm just kidding, it's up to you of course)

Comment (tcezard) [reply eva_sub_cli/__init__.py]: Added now :)

**Pull request merged**